android-lib/src/main/java/com/kinvey/android/Client.java

```diff
@@ -129,14 +129,28 @@ protected Client(HttpTransport transport, HttpRequestInitializer httpRequestInit
         super(transport, httpRequestInitializer, rootUrl, servicePath, objectParser, kinveyRequestInitializer, store,
                 requestPolicy);
         Logger.init(new AndroidLogger());
-        _sharedInstance = this;
+        sharedInstance = this;
         this.context = context;
         cacheManager = new RealmCacheManager(this);
         syncCacheManager = new RealmCacheManager("sync_", this);
     }
 
     public static Client sharedInstance(){
-    	return _sharedInstance;
+    	return sharedInstance;
+    }
+
+    @Override
+    public void setUser(T user) {
```

Comment:
@yuliya-guseva This method should probably be called `setActiveUser()`, to pair it with `getActiveUser()`.